spec/rails_helper.rb

```diff
@@ -1,3 +1,34 @@
+def test_data
+  @studio_1 = Studio.create!(name: "Universal Studios", location: "Hollywood")
```

Comment:
consider adding this data to the seeds file as well so that you can get this data in development